api/src/endpoints/svgThumbnail.js

```diff
@@ -0,0 +1,44 @@
+import express from 'express';
+import fs from 'fs';
+import getSvgThumbnail from 'utils/2d-map';
+import models from 'data/integratedModels.json';
+
+const VALID_MODELS = models.map((m) => m.short_name);
+
+const routes = express.Router();
+const sharp = require('sharp');
+
+routes.get('/:svgName', async (req, res) => {
+  const { svgName } = req.params;
+  const { model, version, width } = req.query;
```

Comment:
const version is unused?

Reply:
no. But it is suggested by Mihail to add it. I guess it might be used in the future.

Reply:
This param is needed in the case when we update the model version and we don't have the older one in the website.

Reply:
If we always keep one version of 2D maps on the website, then we don't need the version parameter even if we updated the model version. It is needed only we are trying to keep 2D maps of multiple model versions. In addition, for the 2D map viewer, it is important that the model version in the svg file, i.e. `data-modelversion`, matches the model version in the yaml file. However, for this thumbnail API, it doesn't matter. It just create a PNG thumbnail of the svg file and don't check the model version in it. Does it make sense @mihai-sysbio @inghylt ?

Reply:
I see. Fine by me as long as there is a note of this in https://github.com/MetabolicAtlas/private-issues/issues/39.